packages/lodestar/src/chain/blocks/importBlock.ts

```diff
@@ -111,12 +112,12 @@ export async function importBlock(chain: ImportBlockModules, fullyVerifiedBlock:
   // Only process attestations in response to an non-prefinalized block
   if (!skipImportingAttestations) {
     const attestations = Array.from(readonlyValues(block.message.body.attestations));
-
+    const rootCache = new altair.RootCache(postState);
```

Comment:
@dapplion getting expected root from forkchoice and compare it here seems to be better in terms of performance, could we do that?

Reply:
How would that look like?

Reply:
`correctHead = ssz.Root.equals(rootCache.getBlockRootAtSlot(data.slot), data.beaconBlockRoot);`
=>
```ts
const protoBlock = forkchoice.getCanonicalBlockAtSlot(data.slot)
correctHead = ssz.Root.equals(fromHexString(protoBlock.blockRoot), data.beaconBlockRoot);
```

it's not 100% equivalent but I think it's faster, we have to implement a cache here to not always go to forkchoice to get block through